docs/breaking-changes.md

```diff
@@ -22,7 +22,7 @@ rejection occurs instead of crashing the process.
 To restore the previous behavior, you can use:
 
 ```js
-process.on('uncaughtException', () => {
+process.on('unhandledRejection', () => {
```

Comment:
Also a quick note that the behavior isn't 100% equivalent due to https://github.com/electron/electron/issues/44174.